src/XamlStyler.dotUltimate/src/dotnet/Plugin.props

```diff
@@ -0,0 +1,17 @@
+<Project>
+  <PropertyGroup>
+    <SdkVersion>2020.1.0</SdkVersion>
```

Comment:
Needs updating in the future. This is the R# SDK being built with, which should follow the Rider version in the gradle files earlier in this PR.

Reply:
Annotated with a comment in code now as well.